sources/packages/forms/src/form-definitions/studentfinancialinformationappeal.json

```diff
@@ -1549,6 +1561,1188 @@
+                    "hidden": false,
+                    "modalEdit": false,
+                    "key": "html89",
+                    "tags": [],
+                    "properties": {},
+                    "conditional": {
+                        "show": "",
+                        "when": null,
+                        "eq": "",
+                        "json": ""
+                    },
+                    "customConditional": "",
+                    "logic": [],
+                    "attributes": {},
+                    "overlay": {
+                        "style": "",
+                        "page": "",
+                        "left": "",
+                        "top": "",
+                        "width": "",
+                        "height": ""
+                    },
+                    "type": "htmlelement",
+                    "input": false,
+                    "placeholder": "",
+                    "prefix": "",
+                    "suffix": "",
+                    "multiple": false,
+                    "defaultValue": null,
+                    "protected": false,
+                    "unique": false,
+                    "persistent": false,
+                    "clearOnHide": true,
+                    "refreshOn": "",
+                    "redrawOn": "",
+                    "tableView": false,
+                    "dataGridLabel": false,
+                    "labelPosition": "top",
+                    "description": "",
+                    "errorLabel": "",
+                    "tooltip": "",
+                    "tabindex": "",
+                    "disabled": false,
+                    "autofocus": false,
+                    "dbIndex": false,
+                    "customDefaultValue": "",
+                    "calculateValue": "",
+                    "calculateServer": false,
+                    "widget": null,
+                    "validateOn": "change",
+                    "validate": {
+                        "required": false,
+                        "custom": "",
+                        "customPrivate": false,
+                        "strictDateValidation": false,
+                        "multiple": false,
+                        "unique": false
+                    },
+                    "allowCalculateOverride": false,
+                    "encrypted": false,
+                    "showCharCount": false,
+                    "showWordCount": false,
+                    "allowMultipleMasks": false,
+                    "id": "eui7cei",
+                    "addons": [],
+                    "hideLabel": true
+                },
+                {
+                    "label": "Do you want to declare your partner's estimated current year income? (If your spouse/common-law partner has had, or anticipates having a significant decrease in income for {{data.calculatedTaxYear}}, you may request to be assessed using your spouse/common-law partner's estimated income.)",
```

Comment:
The content inside the parenthesis is for the blurb text and not meant to be the question itself. 

`(If your spouse/common-law partner has had, or anticipates having a significant decrease in income for {{data.calculatedTaxYear}}, you may request to be assessed using your spouse/common-law partner's estimated income.)`